src/frequenz/sdk/timeseries/logical_meter/_formula_steps.py

```diff
@@ -57,10 +58,7 @@ def apply(self, eval_stack: List[Optional[float]]) -> None:
         """
         val2 = eval_stack.pop()
         val1 = eval_stack.pop()
-        if val1 is None or val2 is None:
-            res = None
-        else:
-            res = val1 + val2
+        res = val1 + val2
```

Comment:
The idea/goal is the MetricFetcher only propagates NaN or 0s (based on the configuration) rather than `None` such us `FormulaStep` doesn't need to deal with `None`